packages/synchro-charts/src/testing/test-routes/widget-test-route.tsx

```diff
@@ -12,20 +15,26 @@ const {
   messageOverrides,
   width = DEFAULT_WIDTH,
   height = DEFAULT_HEIGHT,
+  size,
   axis,
   componentTag,
   annotations,
   viewportStart,
   viewportEnd,
   duration,
   isEditing,
+  associatedStreams,
+  dataStream,
   dataStreams,
   gestures,
   legend,
   tableColumns,
 } = testCaseParameters();
 
-const getSize = (value: number | string): { height: number | string; width: number | string } | undefined => {
+const getSize = (
```

Comment:
we should be parsing the 'test case parameters' in the `testCaseParameters` method - in the same way everything else is parsed.

Reply:
Done